bokehjs/test/integration/regressions.ts

```diff
@@ -4699,4 +4699,68 @@ describe("Bug", () => {
       ctx.drawImage(image, 0, 0, 200, 200)
     })
   })
+
+  describe("in issue #14549", () => {
+    it("doesn't prevent hover action upon bbox change", async () => {
+      function linspace(start: number, stop: number, num: number, endpoint = true) {
+        const div = endpoint ? (num - 1) : num
+        const step = (stop - start) / div
+        return Array.from({length: num}, (_, i) => start + step * i)
+      }
+
+      const n = 1000
+      const x = linspace(0, 20, n)
+      const y = x
+
+      const div = new Div({text: "some text"})
+      const source = new ColumnDataSource({data: {x, y}})
+
+      function hover_cb(_model: HoverTool, options: any) {
+        const {index} = options
+        let idx = []
+        if (typeof index !== "undefined") {
+          idx = index.line_indices
+        }
```

Comment:
```suggestion
        const idx = index?.line_indices ?? []
```

Reply:
Given that `index` is always provided, this should actually be:

```suggestion
        const idx = index.line_indices
```